src/index.ts

```diff
@@ -100,7 +100,7 @@ commander
   .description('Create a script')
   .option(
     '--type <type>',
-    'Creates a new add-on attached to a new Document, Spreadsheet, Presentation, or Form.',
+    'Creates a new Apps Script project attached to a new Document, Spreadsheet, Presentation, or Form.',
```

Comment:
Please revert. These are called 'add-ons' if they are attached to a script.

Reply:
@grant I'm reverting.
Not that I want to argue, but doesn't Clasp addresses all the three types of scripts:

- standalone (including libraries?)
- bound to G Suite Documents (including add)ons)
- Web Apps

If so, reformulating this description would make sense, wouldn't it?

Reply:
Oh, I'd actually just change the comment to include all possibilities:

https://github.com/google/clasp/#examples-2

```
Creates a new Apps Script project attached to a new Document, Spreadsheet, Presentation, Form, or as a standalone script, web app, or API.
```

Reply:
Your wish is my command.